front/static/front/js/deck_edit.js

```diff
@@ -0,0 +1,24 @@
+url_home=get_urls().url_home
+api_deck=get_urls().api_deck
+let start_name=''
```

Comment:
I don't know JS to well, but should you use global variables?

Reply:
you are right, i deleted global var

Reply:
Still exists?

Reply:
ok, I don't use url_home and api_deck in this form from your snapcode